src/components/ZipInput.tsx

```diff
@@ -6,21 +6,27 @@ type ZipInputProps = {
   zip: string;
   setZip: (zip: string) => void;
   showError?: boolean;
+  noLabel?: boolean;
+  autoFocus?: boolean;
 };
 
 function ZipInput({
   children,
   zip,
   setZip,
+  noLabel = false,
+  autoFocus = false,
   showError = false,
 }: PropsWithChildren<ZipInputProps>) {
   const { t } = useTranslation();
 
   return (
     <div className="width-full">
-      <Label htmlFor="zip" className="margin-bottom-1">
-        {t(`common.zipCode`)}
-      </Label>
+      {!noLabel && (
```

Comment:
I think label is required for input components, accessibility-wise. to hide you should add the class “usa-sr-only” (or maybe it’s just “sr-only” but I think it has USA prefix) which will make the label only ‘visible’ to screen readers but not in the actual visible DOM.

Reply:
great call, done 💯